<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsummaryReportDlg</class>
 <widget class="QDialog" name="CsummaryReportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>550</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_savePDF">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>550</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save PDF</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_fromDate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_toDate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_textBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>631</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_fromLBL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_toLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_useAccountNameCkbox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Use Account Name</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
